README.md

```diff
@@ -1,21 +1,21 @@
 # <img width="80" height="50" src="./logo/logo.png" /> Static Land
 
-Specification for common algebraic types in JavaScript
-based on [Fantasy Land](https://github.com/fantasyland/fantasy-land).
+Specification for common algebraic structures in JavaScript
+based on [fantasy-land](https://github.com/fantasyland/fantasy-land).
```

Comment:
Yeah, I'm going back and forth with this. I like how lower case "static-land" looks (except in titles), but if I use lower case "static-land" then "Fantasy Land" next to it looks off.